tests/test_pacer.py

```diff
@@ -634,6 +634,13 @@ def test_district_court_dockets(self):
                                  'district')
         self.run_parsers_on_path(path_root)
 
+    def test_district_court_dockets(self):
+        path_root = os.path.join(TESTS_ROOT, 'examples', 'pacer', 'dockets',
+                                 'fake-rss')
+        self.run_parsers_on_path(path_root,
+                                 requiredFields = ['case_name', 'docket_number']
```

Comment:
Funky indentation/spacing again; needs trailing comma.

Reply:
Err, I don't see the indentation issue? You want a trailer comma in a python list that's all on one line? Why? PEP8 doesn't cal for it? Trailing commas are nice in multiline lists because then diffs don't disturb lines unrelated to the change. But that case is not this case? What am I missing?

Anyhow, `flake8` has things to say, so I'll just make it happy.

Reply:
The paren is on it's own line, but I think the way github is commenting this, you can't necessarily see that? Maybe just move that up, fix the spacing around the `=`, and we're in business here.

Reply:
moot